.gitattributes

```diff
@@ -1,5 +1,6 @@
 apps/i18n/** filter=lfs diff=lfs merge=lfs -text
 dashboard/config/locales/** filter=lfs diff=lfs merge=lfs -text
+dashboard/config/locales/en.yml !text !filter !merge !diff
```

Comment:
I just tested without this new line, and after #56737 I can do this:
```
git checkout staging
git checkout -b seth/test-en-yml-diffing
# Make a small change to the file
echo "  small_change: 'By seth'" >> dashboard/config/locales/en.yml
git diff staging
```

Which gives me this diff:
```
diff --git a/dashboard/config/locales/en.yml b/dashboard/config/locales/en.yml
index 708dbc66dc8..b728dc7acc4 100644
--- a/dashboard/config/locales/en.yml
+++ b/dashboard/config/locales/en.yml
@@ -1434,3 +1434,4 @@ en:
       success_body: "Great news! Your LMS has been registered with Code.org, and you can continue installing and using your LMS integration. You should receive a confirmation email shortly."
       success_header: "Thank you for registering your LMS!"
       register: "Register LMS"
+  small_change: 'By seth'
```

So at least basic diffing has started working, and I tested merges between two branches that change en.yml in different places as well, which also seem to be working:

1. Create branch off staging: `seth/change_en_yml`, and modify top of en.yml, `git diff staging` is:
```
diff --git a/dashboard/config/locales/en.yml b/dashboard/config/locales/en.yml
index 708dbc66dc8..b728dc7acc4 100644
--- a/dashboard/config/locales/en.yml
+++ b/dashboard/config/locales/en.yml
@@ -1434,3 +1434,4 @@ en:
       success_body: "Great news! Your LMS has been registered with Code.org, and you can continue installing and using your LMS integration. You should receive a confirmation email shortly."
       success_header: "Thank you for registering your LMS!"
       register: "Register LMS"
+  small_change: 'By seth'
```

2. Create branch off staging: `seth/change_en_yml_2` and modify bottom of en.yml, `git diff staging` is:
```
diff --git a/dashboard/config/locales/en.yml b/dashboard/config/locales/en.yml
index 708dbc66dc8..b8feed39788 100644
--- a/dashboard/config/locales/en.yml
+++ b/dashboard/config/locales/en.yml
@@ -31,6 +31,7 @@
 
 en:
   errors:
+    another: 'small change by seth'
     messages:
       not_saved: 'Please correct the following errors:'
       too_young: 'This content has age restrictions in place and is not available for younger students.'
```

4. Now try to successfully merge both changes:
```
git checkout seth/change_en_yml_2
git merge seth/change_en_yml
# Auto-merging dashboard/config/locales/en.yml
# Merge made by the 'ort' strategy.
# dashboard/config/locales/en.yml | 1 +
# 1 file changed, 1 insertion(+)
```

5. Finally, verify that both changes are included `git diff staging`:
```
diff --git a/dashboard/config/locales/en.yml b/dashboard/config/locales/en.yml
index 708dbc66dc8..b8baca1c837 100644
--- a/dashboard/config/locales/en.yml
+++ b/dashboard/config/locales/en.yml
@@ -31,6 +31,7 @@
 
 en:
   errors:
+    another: 'small change by seth'
     messages:
       not_saved: 'Please correct the following errors:'
       too_young: 'This content has age restrictions in place and is not available for younger students.'
@@ -1434,3 +1435,4 @@ en:
       success_body: "Great news! Your LMS has been registered with Code.org, and you can continue installing and using your LMS integration. You should receive a confirmation email shortly."
       success_header: "Thank you for registering your LMS!"
       register: "Register LMS"
+  small_change: 'By seth'
```

I should add that none of this worked before #56737, so if the branch you are trying to merge in doesn't have #56737 yet, this might not work until its merged in.